test/cases/migration/index_test.rb

```diff
@@ -0,0 +1,199 @@
+# frozen_string_literal: true
+
+require "cases/helper"
+
+module ActiveRecord
+  module CockroachDB
+    class Migration
+      class IndexTest < ActiveRecord::TestCase
+        # These tests are identical to the ones found in Rails, save for the fact
```

Comment:
Just for my own knowledge: how are you turning off the tests related to transactions here? Did you just remove those lines?

Reply:
the self.use_transactional_tests setting turns off transactions, and also i only copied in the tests that are expected to pass

Reply:
Oh I see. Didn't realize you could just use that built-in line to turn off transactions. Found docs: https://edgeguides.rubyonrails.org/testing.html#testing-parallel-transactions